package.json

```diff
@@ -16,8 +20,7 @@
     "webpack:dev": "MODE=dev webpack-dev-server --mode development --progress",
     "electron:dev": "MODE=dev electron .",
     "start": "concurrently \"BROWSER=none npm run webpack:dev\" \"wait-on http://localhost:9000 && npm run electron:dev\"",
-    "build": "webpack --mode production && electron-builder --windows",
-    "test": "jest"
```

Comment:
Should test command be removed?

Reply:
Of course not, it should be here. Will be fixed.